solutions/izho-14-bank.mdx

```diff
@@ -2,34 +2,184 @@
 id: izho-14-bank
 source: IZhO 2014
 title: Bank
-author: Benjamin Qi
+author: Kevin Sheng
 ---
 
+We do dynamic programming on all subsets of the notes.
+
+Let $\texttt{covered}[S]$ be the maximum prefix of people we can pay off using the notes defined in the subset, and $\texttt{leftover}[S]$ be the leftover amount of money we have after we pay off the prefix of the people.
+
+When transitioning, iterate over all possible "ending" banknotes and see how each of them would fit in when you put them onto the leftovers of the previous subset.
+
```

Comment:
add `## Implementation`

Reply:
this is just convention for the solutions right

Reply:
implementation should be above time complexity